ocaml/xapi/repository_helpers.ml

```diff
@@ -1425,6 +1424,12 @@ let get_ops_of_pending ~__context ~host ~kind =
            (Ref.string_of vm_ref, pending_guidances)
        )
   in
+  let is_vm_applicable ~self = function
+    | `restart_device_model ->
+        Helpers.has_qemu_currently ~__context ~self
+    | `restart_vm ->
+        false
```

Comment:
I think this is to avoid any `restart_vm as mandatory guidance? Maybe better to add a log when ignoring it?